addons/website/static/src/js/website.snippets.editor.js

```diff
@@ -345,7 +352,6 @@ options.registry.parallax = options.Class.extend({
     _refresh: function () {
         _.defer((function () {
             this.$target.data("snippet-view")._rebuild();
-            this._update_target_to_bg();
         }).bind(this));
```

Comment:
Yes. This fixes the issue of Parallax's background being changed when just browsing ( without clicking ) available Background Images from drop-down.